oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -393,7 +393,7 @@ query:
     # abort a query if the total number of data fields queried exceeds the defined threshold.
     maxQueryComplexity: ${SW_QUERY_MAX_QUERY_COMPLEXITY:3000}
     # Allow user add, disable and update UI template
-    enableUpdateUITemplate: ${SW_ENABLE_UPDATE_UI_TEMPLATE:false}
+    enableUpdateUITemplate: ${SW_ENABLE_UPDATE_UI_TEMPLATE:true}
```

Comment:
Why modify the default values? Please confirm if this modification is related to this feature.
If necessary, please modify the corresponding document. (`skywalking/docs/en/setup/backend/Configuration vocabulary.md`)

Reply:
This should not open by default.
I believe this is a local change, please revert.

Reply:
Sorry, this is just for local testing convenience. I will turn it off before the final version

Reply:
The best practice is, you should revert this, and change the local system environment variable to set as true.